prow/cluster/jobs/istio-private/istio/istio-private.istio.release-1.4.gen.yaml

```diff
@@ -11,7 +11,11 @@ postsubmits:
         bucket: istio-private-build
       ssh_key_secrets:
       - ssh-key-secret
-    name: unit-tests_istio_release-1.4_postsubmit_priv
+    labels:
+      preset-enable-ssh: "true"
+      preset-service-account: "true"
+    max_concurrency: 5
+    name: release_istio_release-1.4_postsubmit_priv
```

Comment:
the diff here is interesting, is it not deterministic?

Reply:
I noticed that as well. I can likely sort the array of jobs before writing them to fix this.

Reply:
Added sorting for env vars: https://blog.golang.org/go-maps-in-action#TOC_7.